packages/node/test/unit/ConfigWizard.test.ts

```diff
@@ -78,10 +78,29 @@ const IMPORTED_PRIVATE_KEY =
 
 const OPERATOR_ADDRESS = '0x54d68882d5329397928787ec496da3ba8e45c48c'
 
+const parseStoragePath = (summary: string): string | undefined => {
```

Comment:
I'd suggest to call this `extractStoragePath` rather than `parseStoragePath`.

My reasoning:
1. One "extracts" when they pull out a substring that matches a pattern. That's it.
2. One "parses" when they analyse to understand and, for instance, splits something into smaller pieces, etc.

In other words, `parseStoragePath` would have to take \*a storage path* as an argument because that's what it parses. What we currently have is `parseSummary`.

Reply:
Very good improvement, thanks 👍